app/models/role.rb

```diff
@@ -40,6 +41,13 @@ def includes?(role_name)
     end
   end
 
+  def github_handle=(github_handle)
+    return unless github_handle.present?
+    self.user = User.where('github_handle ILIKE ?', github_handle)
+                    .first_or_initialize(github_handle: github_handle)
+    user.github_import = true
```

Comment:
Something for another PR, but I wondered if this should only be set when a user with that DB couldn't be found

Reply:
Yes, would make sense. Otherwise it should already be set to `true` anyways if I understand correctly... 🤔